js/activity.js

```diff
@@ -5173,13 +5173,22 @@ function Activity() {
             updatePluginObj(obj);
         }
 
+     
+
         // Load custom mode saved in local storage.
         var custommodeData = storage.custommode;
         if (custommodeData != undefined) {
             customMode = JSON.parse(custommodeData);
             console.debug("restoring custom mode: " + customMode);
         }
 
+        // Load rhythm data saved in local storage.
+        var rhythmrulerdata = storage.getItem("rhythmhistory");
+        if (rhythmrulerdata != undefined) {
+            History = rhythmrulerdata;
```

Comment:
History is a JavaScript keyword (the History object) so we probably want to use a different var name here. Also, it should be a var, not a global.